docker-compose.yml

```
@@ -0,0 +1,49 @@
+    volumes:
+      - .:/app
+  mysql:
+    image: mysql:8.0
+    ports:
+      - 3306:3306
+    restart: always
+    environment:
+      - MYSQL_DATABASE=bodytrak_db
+      - MYSQL_ROOT_PASSWORD=MyPassword2024
+    volumes:
+      - mysqldata:/var/lib/mysql     
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)